.github/workflows/base_node_binaries.yml

```diff
@@ -192,13 +188,16 @@ jobs:
           echo "LIBCLANG_PATH=C:\Program Files\LLVM\bin" >> $GITHUB_ENV
           echo "C:\Strawberry\perl\bin" >> $GITHUB_PATH
 
-      # Don't use caches for binary builds. Start from a clean slate.
+      # Don't use caches for binary builds. Start from a clean slate - on release
+      - name: Cache cargo files and outputs
+        if: ${{ ! startsWith(github.ref, 'refs/tags/v') }}
+        uses: Swatinem/rust-cache@v2
 
       - name: Build rust binaries
-        env:
-          CARGO_UNSTABLE_SPARSE_REGISTRY: true
-          RUSTFLAGS: "-C target_cpu=${{ matrix.builds.target_cpu }}"
-          ROARING_ARCH: "${{ matrix.builds.target_cpu }}"
+        # env:
+        #   CARGO_UNSTABLE_SPARSE_REGISTRY: true
+        #   RUSTFLAGS: "-C target_cpu=${{ matrix.builds.target_cpu }}"
+        #   ROARING_ARCH: "${{ matrix.builds.target_cpu }}"
```

Comment:
We need to leave commented out code?

Reply:
I prefer we remove them. 
We have the git history if we want to retrieve them again. 
Commented out code tends to live forever in the code.